app/models/payment_notification/paypal.rb

```diff
@@ -20,7 +20,7 @@
 #  index_payment_notifications_on_type      (type)
 #
 
-class PaymentNotification::Paypal < PaymentNotification
+class PaymentNotification::PayPal < PaymentNotification
```

Comment:
This is breaking the code. You should not change it on the code or, if you are going change it, change it every where.